distributed/actor.py

```diff
@@ -158,8 +157,9 @@ async def run_actor_function_on_worker():
                             kwargs={k: to_serialize(v) for k, v in kwargs.items()},
                         )
                     except OSError:
-                        if self._future:
+                        if self._future and not self._future.done():
                             await self._future
+                            return await run_actor_function_on_worker()
```

Comment:
I'm curious, why this change?

Reply:
We can't run something on an actor until a worker has reported that the actor has been initialised. This branch is for the case that the future representing the actor is still in flight (can't happen twice).

Reply:
OK.  Thank you for the explanation